primitives/src/util.rs

```diff
@@ -82,3 +82,13 @@ impl FromStr for Chain {
         })
     }
 }
+
+impl Chain {
+    #[inline]
+    pub fn is_testnet(self) -> bool {
```

Comment:
IMO inverting logic and calling the method `is_mainnet` would be clearer

Reply:
I do not agree. Right now there is just one mainnet: `Chain::Bitcoin` and one can just test it against it. Testnets are multiple and their number may be growing.

In the future more non-testnets may be added (for instance prime). But they are also not "mainnets". Thus since we need to test against testnet/non-testnet - we need method named `is_testnet` and not `is_mainnet`.

Reply:
IMO `is_testnet` is confusing since there is a specific network called "testnet". Maybe we could call it `is_test_network`?

Reply:
Agree that it is confusing. Probably better `is_test` or `is_no_value`?

BTW the full name for the testnet is `testnet3`. Testnet is not running anymore.

Reply:
I did `is_test_chain` (since the name of the type is `Chain` and not `Network`). Also seems not to conflict with "testnet"